src/components/NewDraftWizard/WizardComponents/WizardSavedObjectsList.js

```diff
@@ -70,35 +69,41 @@ const WizardSavedObjectsList = ({ submissions }: WizardSavedObjectsListProps): R
     submissionTypes.length > 0 &&
     submissionTypes.map(submissionType => ({
       submissionType,
-      submittedItems: submissions.filter(obj => obj.tags.submissionType && obj.tags.submissionType === submissionType),
+      items: objects.filter(obj => obj.tags.submissionType && obj.tags.submissionType === submissionType),
     }))
 
-  const displaySubmissionType = (submission: {
-    submissionType: string,
-    submittedItems: Array<ObjectInsideFolderWithTags>,
-  }) => {
+  const draftObjects = [
+    {
+      submissionType: "Draft",
+      items: objects.filter(object => object.schema.includes("draft-")),
+    },
+  ]
+
+  const listItems = draftObjects[0].items.length ? draftObjects : groupedSubmissions
+
+  const displaySubmissionType = (submission: { submissionType: string, items: Array<ObjectInsideFolderWithTags> }) => {
     switch (submission.submissionType) {
       case ObjectSubmissionTypes.form:
-        return submission.submittedItems.length >= 2 ? "Forms" : "Form"
+        return submission.items.length >= 2 ? "Forms" : "Form"
       case ObjectSubmissionTypes.xml:
-        return submission.submittedItems.length >= 2 ? "XML files" : "XML file"
+        return submission.items.length >= 2 ? "XML files" : "XML file"
       default:
-        return ""
+        return submission.items.length >= 2 ? "drafts" : "draft"
     }
   }
 
   return (
     <div className={classes.objectList}>
-      {groupedSubmissions &&
-        groupedSubmissions.map(group => (
+      {listItems &&
+        listItems.map(group => (
           <Box pt={0} key={group.submissionType}>
             <CardHeader
               title={`Submitted ${formatDisplayObjectType(objectType)} ${displaySubmissionType(group)}`}
```

Comment:
not sure if this is where it is, but looks like it, maybe for drafts we want other label than `Submitted`

![image](https://user-images.githubusercontent.com/47524/129674208-80393f6b-b390-4e78-8e17-c38888481c29.png)

maybe `<object> Drafts` e.g. `Sample Drafts` and the colour of the background of title could be different to emphasize that this is different than the submitted objects

Reply:
or we can have something like
- `Sample Submitted Forms`
- `Sample Submitted XML files`
- `Sample Drafts`